### PR TITLE
Update pytest-mpl to 0.5

### PR DESCRIPTION
**Pull request by astrofrog**
Is this the right way to update a package's version?

### CODE REVIEW

**Comment by astrofrog** [comment]
@dopplershift - I wonder if it would make sense to add me as a maintainer of this feedstock? (since I'm the lead dev of pytest-mpl). If you agreed to that, should I add my username in the recipe-maintainers section?

**Comment by ocefpaf** [comment]
> @dopplershift - I wonder if it would make sense to add me as a maintainer of this feedstock? (since I'm the lead dev of pytest-mpl).

Yes please!

> If you agreed to that, should I add my username in the recipe-maintainers section?

Exactly you can se a PR adding your GitHub handle and you will be automatically added to the @conda-forge/pytest-mpl team. That should give you commit rights to this repository.

Feel free to browse the list of [feedstocks](https://conda-forge.github.io/feedstocks.html) and add yourself as a maintainer of those that interest you.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 package:
   name: pytest-mpl
   version: {{ version }}
 
 source:
   fn: pytest-mpl-{{version}}.tar.gz
-  url: https://pypi.python.org/packages/source/p/pytest-mpl/pytest-mpl-{{version}}.tar.gz
-  md5: 45912acdd8ae63358711de431ac076ec
+  url: https://pypi.python.org/packages/cb/db/a7b8c2bb859b4f021e349adbebe6968036dd6f236a98be795c1fe621d340/pytest-mpl-{{version}}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Can you change that ugly new PyPI URL to `https://pypi.io/packages/source/p/pytest-mpl/pytest-mpl-{{ version }}.tar.gz`please?

Comment (astrofrog) [inline recipe/meta.yaml]: Fixed!

**Comment by dopplershift** [comment]
@astrofrog Absolutely happy to add you as a maintainer--I'm sure you'll be much better about staying on top of the latest releases. 😄

**Pull request merged**